dlc/src/lib.rs

```diff
@@ -65,14 +68,36 @@ const ENABLE_LOCKTIME: u32 = 0xfffffffe;
 /// Represents the payouts for a unique contract outcome. Offer party represents
 /// the initiator of the contract while accept party represents the party
 /// accepting the contract.
+#[derive(PartialEq, Debug, Clone)]
 pub struct Payout {
     /// Payout for the offering party
     pub offer: u64,
     /// Payout for the accepting party
     pub accept: u64,
 }
 
+#[derive(PartialEq, Debug)]
+/// Representation of a set of contiguous outcomes that share a single payout.
+pub struct RangeOutcome {
```

Comment:
nit: I'm not sure this is the most descriptive name, in bitcoin-s we use `ConstantPayoutRange` I think

Reply:
Went with `RangePayout`, `ConstantPayoutRange` felt a bit long.